include/sacn/receiver.h

```diff
@@ -111,17 +129,43 @@ typedef struct SacnLostSource
  * @{
  */
 /*! Filter preview data. If set, any sACN data with the Preview flag set will be dropped for this
- *  universe and sources sending only Preview data will not be tracked. */
+ *  universe but sources sending only Preview data will still trigger a SacnSourcesFoundCallback(). */
```

Comment:
This seems like it creates an opportunity to misuse the API, since the SacnSourcesFoundCallback also passes back the initial DMX data, so someone could act on the first packet even if subsequent ones don't come in UniverseData after that.

Reply:
That's what we do.  I've added a bit more description to the comment:
 * In the rare case where the source is only sending preview packets and SACN_RECEIVER_OPTS_FILTER_PREVIEW_DATA is set,
 * this callback will be be called with a values_len of 0 and values_were_preview set to true.

I did it this way because I wanted to allow a preview source send non DMX packets to the receiver, and we stated that the receiver would only receive data if it was notified of the source.

Reply:
This is interesting actually, because the Preview_Data flag is at the sACN framing layer, theoretically independent of any start code. I suppose the same source could multiplex DMX (null start code) packets with the Preview bit set, with non-DMX packets without the preview bit set. Would the library consider these to be the same source?

Reply:
Yes.  That's why I changed things around so that data and per-address are bundled together in SourcesFound, and why the merge_receiver differentiates "Merged data" from "NonDmxData"

Reply:
And @ChristianReese , if some of this isn't clear from the todos or Jama requirements, we can work on spelling it out further.